nio/block/base.py

```diff
@@ -28,7 +28,7 @@ class Base(PropertyHolder, CommandHolder, Runner):
     id = StringProperty(title="Id", visible=False, allow_none=False)
     name = StringProperty(title="Name", visible=False, allow_none=True)
```

Comment:
Good opportunity here to remove `visible=False` here

Reply:
I'm not exactly sure what the implications of that would be (but granted, I've been out of the loop over the last week). I would think it would mainly have designer implications but it might be best to keep that convo separate

Reply:
^^ That would require an update to the designer. Right now we have the `name` hardcoded in, so removing `visible=False` here without any updates to the designer would show 2 name fields then. 

There would have to be some logic around deciding whether or not the name property exists as to if to show it or not.

I'm fine with having that be a discussion for another day.